src/js/components/account/visualizations/time/chart/BarChartStacked.jsx

```diff
@@ -363,7 +363,7 @@ ${xAxis.items[0].label} to ${xAxis.items[xAxis.items.length - 1].label}.`;
                 <svg
                     className="bar-graph"
                     width={this.props.width}
-                    height={this.props.height + 20}>
+                    height={this.props.height + 40}>
```

Comment:
there's a comment at line 358 that refers to this 20 that you changed, so you should change the comment too